egs/babel_multilang/s5/local/nnet3/run_multilingual_bnf.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# This script trains a multilingual model using 6 layer TDNN + Xent 
+# with 42 dim bottleneck layer in fifth layer for Georgian.
+# The lang_list contains 10 closest fullLP langs to Georgian + fullLP Georgian.
+# Then it extracts bottleneck features for input language "lang" and 
+# train SAT model using these feautures.
+
+# Copyright 2016  Pegah Ghahremani
+# Apache 2.0
+
+#This yields approx 70 hours of data
+# this script generates bottleneck features from multilingual model
+# trained on list of languages and dump the bnf for specific language L.
+set -e           #Exit on non-zero return code from any command
+set -o pipefail  #Exit if any of the commands in the pipeline will
+                 #return non-zero return code
+. conf/common_vars.sh || exit 1;
+
+set -u           #Fail on an undefined variable
+bnf_train_stage=-100 # the stage variable used in multilingual bottleneck training.
+stage=1
+num_archives=20
+speed_perturb=true
+multidir=exp/nnet3/multi_bnf_10_close_lang_plus_grg
+global_extractor=exp/multi/nnet3/extractor
+lang_list_for_grg=(GRG LIT MONG TUR KAZ KUR PSH SWA TOK IGBO DHO)
```

Comment:
Instead of inventing your own codes, I'd suggest using the ids from babel, which is an established practice... i.e 404-georgian, 304-lithuanian and so on.
It does not help anything if you make the codes short, you only obfuscate things